README.md

```diff
@@ -4,6 +4,33 @@ This is a repository for talks given by Gonum developers about Go and Gonum.
 
 - [Introduction to Go and Concurrency](http://talks.godoc.org/github.com/gonum/talks/2015/intro_concurrency.slide)
 
+## Running presentations locally
+
+Some slides in stacks may include playable code examples that depend on github.com/gonum tree imports, or other
+packages outside the standard library. To play these examples the presentations must be served locally using the
+present tool.
+
+The present tool is easily installed using go get:
+
+```
+go install golang.org/x/tools/cmd/present
```

Comment:
i'd spell `go get` as `go get` and not `go install` :)

ie:
```sh
sh> go get -u golang.org/x/tools/cmd/present
```

(these are presumably instructions for beginners with no real grasp on the differences b/w `install` and `get`)